qir/qat/Passes/ValidationPass/TargetQisConfiguration.cpp

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "qir/qat/Passes/ValidationPass/TargetQisConfiguration.hpp"
+
+#include "qir/qat/Commandline/ConfigurationManager.hpp"
+#include "qir/qat/QatTypes/QatTypes.hpp"
+
+namespace microsoft::quantum
+{
+
+void TargetQisConfiguration::setup(ConfigurationManager& config)
+{
+    config.setSectionName("Validation configuration", "");
+    /// CLI and config accessible
```

Comment:
A doc comment in the body? Consider
```suggestion
    // CLI and config accessible
```